Automattic/wp-calypso
# Remove unused StatsMain component

This PR removes an unnecessary level of abstraction from the Calypso codebase.

`StatsMain` has been rendered redundant by https://github.com/Automattic/wp-calypso/pull/6834

### Testing instructions

Click on the Stats pages and verify that they load as they did before. Also verify that the Stats First View layer is displayed when you navigate to the stats page. If you dismissed the First View before, you can reset it by visiting `/stats/reset-first-view`.

Test live: https://calypso.live/?branch=remove/unused-stats-main-component
